Source/Charts/Data/Implementations/Standard/PieChartDataEntry.swift

```diff
@@ -19,12 +19,21 @@ open class PieChartDataEntry: ChartDataEntry
         super.init()
     }
     
+
+    /// - Parameters:
+    ///   - value: The value on the y-axis
+    @objc public init(value: Double)
+    {
+        super.init(x: .nan, y: value)
```

Comment:
why we use NAN instead of 0?

Reply:
Because `x` is invalid for PieChartDataEntry and shouldn't be used. If it is being incorrectly used, this will more likely trap.